build-tools/xabootstrap/xabootstrap/FilesToRemove.md

```diff
@@ -0,0 +1,12 @@
+# Files to remove
```

Comment:
This feels like material for `README.md`.  Alternatively, more details need to be here.

For example, what constitutes "the migration"?  How will this interact with checking out different branches?

Reply:
It's just notes to myself, so that I don't forget to remove stuff later on (there's a LOT of it around).